internal/cli/cmd/get/get.go

```diff
@@ -34,20 +38,30 @@ kargo get promotions --project=my-project --stage=my-stage
 `),
 	}
 
+	cmdOpts := getOptions{}
+
+	cmdOpts.addFlags(cmd)
+
 	// Register subcommands.
-	cmd.AddCommand(newGetCredentialsCommand(cfg, streams))
-	cmd.AddCommand(newGetFreightCommand(cfg, streams))
-	cmd.AddCommand(newGetProjectsCommand(cfg, streams))
-	cmd.AddCommand(newGetPromotionsCommand(cfg, streams))
-	cmd.AddCommand(newGetStagesCommand(cfg, streams))
-	cmd.AddCommand(newGetWarehousesCommand(cfg, streams))
+	cmd.AddCommand(newGetCredentialsCommand(cfg, streams, cmdOpts))
+	cmd.AddCommand(newGetFreightCommand(cfg, streams, cmdOpts))
+	cmd.AddCommand(newGetProjectsCommand(cfg, streams, cmdOpts))
+	cmd.AddCommand(newGetPromotionsCommand(cfg, streams, cmdOpts))
+	cmd.AddCommand(newGetStagesCommand(cfg, streams, cmdOpts))
+	cmd.AddCommand(newGetWarehousesCommand(cfg, streams, cmdOpts))
```

Comment:
As the values of `cmdOpts` are set at runtime, the constructors should take a pointer I think. Otherwise the propagated value won't be available to the downward commands.

Reply:
Test drove and this observation is correct.

Reply:
Ow that's odd I gave it a try and it did seem to work fine but ill give it a second look

Reply:
well actually properly rebuilding the cli to test will probably help, have pushed a fix to amend.

Reply:
fwiw, when actively working on the cli, I do this:

```
alias kargo='go run cmd/cli/*.go'
```

Reply:
ow that is probably better for me to do than what I have now